client/src/components/FormInput/DatePicker/datepicker.css

```diff
@@ -0,0 +1,27 @@
+.left input {
```

Comment:
I notice that some styles are present in more than one class, we can consolidate by moving them into a common class to avoid repeating ourselves